behave/features/alternative_help.feature

```diff
@@ -15,4 +15,7 @@ Scenario: Create user and out of scope case
     AND I select the diagnosis <category> and click next <number> times
     | category                                                | number |
     | Crime                                                   | 2      |
-    THEN I get an OUTOFSCOPE decision
\ No newline at end of file
+    THEN I get an OUTOFSCOPE decision
```

Comment:
`THEN` and other keywords are in capitals, other feature files aren't. My IDE complains if it's in caps. Does it cause any issues for you?

Reply:
It doesn't matter whether they are capitalised or not in terms of running the tests. I prefer lower case but others seem to like upper.